bundles/http/Citrix/00000019-0000-0000-0000-000000000000/47a6eb95-6f9d-44df-8be7-2322ada5770c.js

```diff
@@ -0,0 +1,864 @@
+const limit = 100 // limit number of records per page for pagination
+const maxNoOfPages = 100000 //  limit number of pages for pagination
+
+function calculateNumberOfPages(totalrecords) {
+	if (Number.isInteger(totalrecords)) {
+		let totalpages = totalrecords / limit
+		if (totalpages > maxNoOfPages) {
+			console.log(`Warning: Number of pages for pagination exceed set value ${maxNoOfPages}. Total number of pages needed: ${totalpages}`)
+			totalpages = maxNoOfPages
+		}
+		return totalpages
+	}
+	else {
+		console.log(`Warning: unable to calculate number of pages, total records returs as: ${totalrecords}`)
+		return 0
+	}
```

Comment:
nit: the final `else` becomes unnecessary
```suggestion
	console.log(`Warning: unable to calculate number of pages, total records returs as: ${totalrecords}`)
	return 0
```